src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -224,6 +224,11 @@ public void waitForElementToBeClickable(WebElement element) {
         WebDriverWait wait = new WebDriverWait(browser.driver, TestProperties.TEST_TIMEOUT);
         wait.until(ExpectedConditions.elementToBeClickable(element));
     }
+    
+    public void waitForElementToBeClickable(String elementId) {
+        WebDriverWait wait = new WebDriverWait(browser.driver, TestProperties.TEST_TIMEOUT);
+        wait.until(ExpectedConditions.elementToBeClickable(browser.driver.findElement(By.id(elementId))));
+    }
```

Comment:
Let's not duplicate the method: just call the pre-defined `waitForElementToBeClickable`. At the very least make use of the previous method here.